core/src/main/java/gyro/core/scope/RootScope.java

```diff
@@ -109,10 +112,25 @@
     private final Boolean inWorkflow;
 
     public RootScope(String file, FileBackend backend, RootScope current, Set<String> loadFiles) {
-        this(file, backend, current, loadFiles, null);
+        this(file, backend, null, current, loadFiles);
     }
 
-    public RootScope(String file, FileBackend backend, RootScope current, Set<String> loadFiles, Boolean inWorkflow) {
+    public RootScope(
+        String file,
+        FileBackend backend,
+        RemoteStateBackend stateBackend,
```

Comment:
Do you think renaming them would be better for readability
 backend is the local state backend I guess
So either `backend` and `remoteBackend` or `stateBackend` and `remoteStateBackend`
Thoughts ??

Reply:
Renamed to `remoteStateBackend` as only state is stored remotely. `backend` was left unchanged as it can store state as well as other files like `init.gyro`.